openvidu-components-angular/projects/openvidu-angular/src/lib/lang/ja.json

```diff
@@ -1,19 +1,19 @@
 {
 	"ADMIN": {
 		"LOGIN": "ログイン",
-		"SECRET": "ひみつ",
-		"SECRET_REQURED": "秘密が必要です",
+		"SECRET": "パスワード",
+		"SECRET_REQURED": "パスワードが必要です",
 		"DASHBOARD": "ダッシュボード",
-		"NO_RECORDINGS": "録音はありません",
-		"SEARCH": "録音を検索する",
-		"DATE": "日にち",
-		"DURATION": "間隔",
+		"NO_RECORDINGS": "録画はありません",
+		"SEARCH": "録画を検索する",
+		"DATE": "日付",
+		"DURATION": "長さ",
 		"SIZE": "サイズ",
-		"STATUS": "状態",
+		"STATUS": "ステータス",
 		"NAME": "名前",
 		"SESSION": "セッション",
 		"OUTPUT": "出力モード",
-		"POWERED_BY": "搭載"
+		"POWERED_BY": "Powered by"
```

Comment:
There's no appropriate translation for `powered by` in japanese, instead we generally use `powered by`.